### DISCUSSION TITLE
QuickLook as preview handler for Windows Explorer

### DISCUSSION THREAD

**Discussion by Gregoras** [QL-Win/QuickLook#1247] [Ideas]
Hi. I want to ask you politely if you would consider to add to QuickLook functionality to acting as general preview handler for File Explorer. You know, in Explorer Preview Pane. If QuickLook resident permanently in memory anyway (it's not accusation, QL is extremely small, on Windows 98 I had more resource havy programs in tray), It would be nice if it also served as support for Explorer.

**Comment by JDFandango** [comment]
I agree that this would be a nice addition.

**Comment by abbasou** [comment]
Attempting to resurrect this topic. Implementing QL as the default preview handler in any given file explorer would be amazing. It's wide compatibility, extensibility, and performance would make a great experience as a dedicated preview pane.

**Reply by emako** [reply]
No plans for now..
